### ISSUE TITLE
axel will truncate Header when header is long

### ISSUE THREAD

**Issue by androidfans** [axel-download-accelerator/axel#352]
Hi, I have noticed that axel will truncate header, that will cause failed to downlaod some long cookie file

below is short header, that can be transfered normal
<img width="1906" alt="1" src="https://user-images.githubusercontent.com/6452650/119604914-766db980-be22-11eb-83a1-e0eb9f22f3ce.png">
below is long header, that will be truncated
<img width="1905" alt="2" src="https://user-images.githubusercontent.com/6452650/119604917-779ee680-be22-11eb-9c19-ee1cf5642d26.png">
below 2 is curl, which will send all header normally
<img width="1903" alt="3" src="https://user-images.githubusercontent.com/6452650/119604921-78377d00-be22-11eb-8023-2e1010592604.png">
<img width="1910" alt="4" src="https://user-images.githubusercontent.com/6452650/119604924-7968aa00-be22-11eb-9d24-c1b51c47d52e.png">

wget can also send all this header 
<img width="1919" alt="6" src="https://user-images.githubusercontent.com/6452650/119605395-4f63b780-be23-11eb-8704-d1fdf1d408b6.png">

**Comment by ismaell** [comment]
Yes, this is a known limitation, axel can't handle requests larger than 2KB.
